dashboard/app/helpers/levels_helper.rb

```diff
@@ -20,6 +20,22 @@ def url_from_path(path)
     "#{root_url.chomp('/')}#{path}"
   end
 
+  def create_channel(data = {}, src = nil)
```

Comment:
Do we have any sort of equivalent to jsdoc for Ruby? If not, I wonder if we should. In particular, it seems like it would be nice to describe what the expectations for these params are. Based on callers, it looks like src is a channel_id or nil (would it make more sense to call this variable channel_id?) and that data is empty string or hidden:true. Does it make more sense to just have this param be a boolean named hidden with a default of false?